src/Point/Components/PointMachine/PointMachine.cs

```diff
@@ -0,0 +1,14 @@
+
+
+namespace EulynxLive.Point.Components
+{
+    public class PointMachine
+    {
+        public PointMachineState state { get; private set; }
```

Comment:
What is the purpose of this class? Can't we use PointMachineState directly?

Reply:
I have removed the class, my other implementation is somewhat more complex and would take a while yet to finish.
I will keep the branch saved locally to use it later when it comes the time.